tests/Unit/PointwiseFunctions/Hydro/Test_QuadrupoleFormula.cpp

```diff
@@ -0,0 +1,42 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <cstddef>
+#include <string>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Structure/ObjectLabel.hpp"
+#include "Framework/CheckWithRandomValues.hpp"
+#include "Framework/SetupLocalPythonEnvironment.hpp"
+#include "Helpers/DataStructures/DataBox/TestHelpers.hpp"
+#include "PointwiseFunctions/Hydro/QuadrupoleFormula.hpp"
+
+namespace hydro {
+
+SPECTRE_TEST_CASE("Unit.PointwiseFunctions.Hydro.QuadrupoleFormula",
+                  "[Unit][Hydro]") {
+  pypp::SetupLocalPythonEnvironment local_python_env(
+      "PointwiseFunctions/Hydro/");
```

Comment:
[random line] Can you add tests for the simple and compute tags? See: https://github.com/sxs-collaboration/spectre/blob/270b1de54b06e9d46c0784f33259b4b324449c80/tests/Unit/Evolution/DgSubcell/Test_Tags.cpp#L60-L61C1 and https://github.com/sxs-collaboration/spectre/blob/270b1de54b06e9d46c0784f33259b4b324449c80/tests/Unit/Evolution/DgSubcell/Test_Tags.cpp#L105-L107 for examples. You'll need to include https://github.com/sxs-collaboration/spectre/blob/270b1de54b06e9d46c0784f33259b4b324449c80/tests/Unit/Evolution/DgSubcell/Test_Tags.cpp#L41

The tests for the compute tags will need to be in the GRMHD system test directory